### PR TITLE
Split DM and admin passwords

### PR DESCRIPTION
**Pull request by avisiedo**
Split PASSWORD to allow to inject secrets for:
- IPA_DM_PASSWORD for the Directory Management password.
- IPA_ADMIN_PASSWORD for the admin password.

This will allow to specify them independently.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,11 +19,17 @@ pass information through the command line. The content could be something like
 the below:
 
 ```raw
-PASSWORD = MyAdminPassword123
+IPA_ADMIN_PASSWORD = Secret123
+IPA_DM_PASSWORD = Secret123
```
Comment (tiran) [inline README.md:23]: Let's use a different password for directory manager in the examples.

```suggestion
IPA_DM_PASSWORD = DMSecret123
```

File: init/ocp4.inc.sh [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+#
+# This need to get a OTP by:
+#
+# ```sh
+# kinit admin
+# ipa host-add replica.example.com --random
+# ```
+#
+# Reference: https://www.adelton.com/docs/idm/replicate-your-identity-management
+IPA_ENROLLMENT_OTP="${IPA_ENROLLMENT_OTP:-${PASSWORD}}"
```
Comment (tiran) [inline init/ocp4.inc.sh:20]: The handling of password vs. OTP is not entirely correct. I suggest that you remove the OTP code for now until we can test this.

**Review by tiran — COMMENTED** [review]
I would leave the OTP part out for now. The ipa-replica-install behaves differently than ipa-server installer.

The server install command takes a ``--ds-password`` and ``--admin-password`` to create the initial passwords.

The replica installer takes a principal (username) password or admin password **or** a password without principal for OTP enrollment. Mixing and matching principal, admin password and OTP password doesn't make much sense.

File: init/ocp4.inc.sh (hunk 2 above)
Comment (avisiedo) [reply init/ocp4.inc.sh]: Thanks mate! updated

**Review by tiran — APPROVED** [review]
LGTM!

Finally works for me

**Pull request merged**